tests/testthat/test-tabPanel.R

```diff
@@ -1,4 +1,87 @@
+# tabsetPanel() et al. use p_randomInt() to generate ids (which uses withPrivateSeed()),
+# so we need to fix Shiny's private seed in order to make their HTML output deterministic
+navlist_panel <- function(...) {
 
+  withPrivateSeed(set.seed(100))
+  navlistPanel(...)
+}
+navbar_page <- function(...) {
+  withPrivateSeed(set.seed(100))
+  navbarPage(...)
+}
+tabset_panel <- function(...) {
+  withPrivateSeed(set.seed(100))
+  tabsetPanel(...)
+}
+
+expect_snapshot2 <- function(...) {
+  if (getRversion() < "3.6.0") {
+    skip("Skipping non-deterministic snapshots on R < 3.6")
```

Comment:
This message confused me for a bit -- I think the default RNG method was changed in 3.6, but it was still deterministic.

```suggestion
    skip("Skipping snapshots on R < 3.6 because of different RNG method")
```